src/bsp/lighting/types.rs

```diff
@@ -1,5 +1,8 @@
 //! Data types split off from the main lighting module for organization.
-
+#![expect(
```

Comment:
Using `expect` means that the CI will automatically tell us when this is no longer needed, probably in 0.17 :)